setup.py

```diff
@@ -3,12 +3,19 @@
 
 from setuptools import setup, Extension
 
+with open("README.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
 
 setup(
+    name="rabbitizer",
+    version="1.0.0",
+    author="Decompollaborate",
+    description="MIPS instruction decoder",
+    long_description=long_description,
```

Comment:
Are those required? 

I defined those on `setup.cfg`, and I think that's the preferred way to do it from what I read, I could be wrong tho